server/bundler/plugin.js

```diff
@@ -29,11 +29,11 @@ ChunkFileNames.prototype.apply = function( compiler ) {
 					this.indent( [
 						"script.onerror = script.onload = script.onreadystatechange = null;",
 						"delete installedChunks[ chunkId ];",
-						"window.__chunkErrors[ " + JSON.stringify( chunkMaps.name ) + "[chunkId]||chunkId ]=new Error();",
+						"window.__chunkErrors[ " + JSON.stringify( chunkMaps.name, null, '  ' ) + "[chunkId]||chunkId ]=new Error();",
```

Comment:
the `null` is the custom formatter. you have to pass it to get at the third arg, which is the string to use when pretty printing. Dropped it all and went back to plain JSON printing. This was really just for debugging. Makes the JSON easier to read.